app/views/prospective_members.html

```diff
@@ -1 +1,33 @@
-<div>prospective members</div>
\ No newline at end of file
+<div class="centered-content padded-content">
+	<script type="text/ng-template" id="ProspectiveMembersSurvey.html">
```

Comment:
Scripts should never be used like this in angular. THis is a directive, which should have its own HTML file in app/directives